### PR TITLE
Rename beta label on the usage page

### PR DESCRIPTION
**Pull request by gtsiolis**
## Description

Following up from https://github.com/gitpod-io/gitpod/pull/12412#issuecomment-1246678522, since https://github.com/gitpod-io/website/pull/2715 is now merged, this will rename the _Beta_ label on the usage page to _Early Access_.

## How to test

1. Enable usage-based pricing for a team.
2. Go to team usage page.
3. Notice the new Early Access label on the left sidebar. 👀 

## Release Notes

```release-note
NONE
```

## Werft options:

- [X] /werft with-preview
- [X] /werft with-payment

### CODE REVIEW

**Comment by gtsiolis** [comment]
Rebased[[1](https://github.com/gitpod-io/gitpod/pull/12993#event-7394341551)] as the [previous build](https://werft.gitpod-dev.com/job/gitpod-build-gt-rename-beta-label-on-the-usage-page.0) failed. 🍀

**Review by svenefftinge — APPROVED** [review]

**Comment by jankeromnes** [comment]
The build failed with:

```
error An unexpected error occurred: "https://registry.npmjs.org/jest-leak-detector/-/jest-leak-detector-26.6.2.tgz: Request failed \"503 Service Unavailable\"".
```

This sounds like a temporary incident on the NPM registry. Simply trying again should fix it:

/werft run

**Comment by sagor999** [comment]
/hold
to unblock merge queue. (build is failing for this PR)

**Comment by gtsiolis** [comment]
I always found obscure why the merge queue is getting blocked and how holding or unholding a PR helps!

**Comment by gtsiolis** [comment]
Rebased[[1](https://github.com/gitpod-io/gitpod/pull/12993#event-74026456081)] as the [previous build](https://werft.gitpod-dev.com/job/gitpod-build-gt-rename-beta-label-on-the-usage-page.2) failed. 🍀

**Pull request merged**
